packages/perseus/src/widgets/interactive-graphs/mafs-graph.tsx

```diff
@@ -1,3 +1,14 @@
+/**
+ * Render the Mafs graph with the specified background and graph elements.
+ *
+ * Render order (back to front):
+ * - Grid
+ * - Axis Ticks, Axis Arrows, and Axis Labels
+ * - Locked Figures
+ * - Locked Labels
+ * - Protractor
+ * - Interactive Graph Elements
+ */
```

Comment:
Thanks for adding this comment! Makes me wonder if maybe each of the layers should be its own component, so the code can be self-documenting, e.g.:

```ts
<GridLayer state={state} {...props} />
<AxisLayer state={state} {...props} />
<LockedFiguresLayer state={state} {...props} />
<LockedLabelsLayer state={state} {...props} />
...
```

Maybe passing the state and props down explicitly wouldn't even be needed, since I think we have the graph context for that purpose? I don't remember exactly what's in there.

Anyway, no action needed. This refactoring could be done separately. I might just do it today.

Reply:
That would make sense to me!